getCMS.js

```diff
@@ -0,0 +1,62 @@
+var fetch = require("node-fetch");
```

Comment:
this is outdated - should also rename to "fetchEvents.js" and also add a "fetchAllCMS.js"